src/Cli/CreateInvoicesForPreviouslyPlacedOrdersCommand.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\InvoicingPlugin\Cli;
+
+use Sylius\Component\Core\Repository\OrderRepositoryInterface;
+use Sylius\InvoicingPlugin\Creator\InvoicesForPreviouslyPlacedOrdersCreatorInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class CreateInvoicesForPreviouslyPlacedOrdersCommand extends Command
+{
+    /** @var InvoicesForPreviouslyPlacedOrdersCreatorInterface */
+    private $invoiceForPreviouslyPlacedOrdersCreator;
+
+    /** @var OrderRepositoryInterface */
+    private $orderRepository;
+
+    public function __construct(
+        InvoicesForPreviouslyPlacedOrdersCreatorInterface $invoiceForPreviouslyPlacedOrdersCreator,
+        OrderRepositoryInterface $orderRepository
+    ) {
+        parent::__construct('sylius-invoicing:create-invoices-for-previously-placed-orders');
```

Comment:
`sylius-invoicing:create-invoices-for-previously-placed-orders`... Hey Macarena! 💃 Why not just `sylius-invoicing:generate-invoices`?

Reply:
The same with the command name